index.html

```diff
@@ -14,6 +16,77 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <header>
+    <nav>
+      <a href="">Portfolio</a>
+      <a href="https://adventurini.github.io/portfolio-website/">Dev Projects</a>
+      <a href="">Blog</a>
+      <a href="">Contact</a>
+    </nav>
+
+  </header>
+  <div class="container">
+    <div class="heading">
+  <h1>ANTHONY VENTURINI
+  <div class="border">
+
+  </div>
+</h1>
+  <h2>FULL STACK DEVELOPER</h2>
+  <div class="table">
+  <ul id="horizontal-list"><li>HTML</li> <li>CSS</li> <li>JAVASCRIPT</li> <li>REACT</li>  <li>DJANGO</li> </ul>
```

Comment:
Nice way to keep your code a little more clean here.
```suggestion
  <ul id="horizontal-list"><li>HTML</li> <li>CSS</li> <li>JAVASCRIPT</li> <li>REACT</li>  <li>DJANGO</li> </ul>
```